core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -158,6 +162,21 @@ public static Block createRandomBlockForType(Type type, int positionCount, float
         if (type == VARCHAR) {
             return createRandomStringBlock(positionCount, nullRate, MAX_STRING_SIZE);
         }
+        if (type instanceof CharType) {
```

Comment:
Why is there no `CharType.CHAR` field like with the other types?

Reply:
`CharType` is parametrized (`length` param) + it does not have `UNBOUNDED_LENGTH` like `VarcharType`